src/OrleansRuntime/ReminderService/LocalReminderService.cs

```diff
@@ -400,7 +477,7 @@ private bool TryStopPreviousTimer(GrainReference grainRef, string reminderName)
         private async Task DoResponsibilitySanityCheck(GrainReference grainRef, string debugInfo)
         {
             if (status != ReminderServiceStatus.Started)
-                await startedTask.Task;
+                await startedTask.Task.WithTimeout(InitialLoadMaxWaitTimeForUpdates);
```

Comment:
Oh,  I miss understood. I thought you were suggesting that I wait longer, until no retries are left.  Good,  I will change the exception.